lib/location-parser.ts

```
@@ -11,105 +11,95 @@ const US_STATES = [
-  'nh': 'new hampshire',
-  'me': 'maine',
-  'mt': 'montana',
-  'ri': 'rhode island',
-  'de': 'delaware',
-  'sd': 'south dakota',
-  'nd': 'north dakota',
-  'ak': 'alaska',
-  'vt': 'vermont',
-  'dc': 'district of columbia'
+  ny: 'new york', ca: 'california', tx: 'texas', fl: 'florida', ga: 'georgia', il: 'illinois', pa: 'pennsylvania', oh: 'ohio', nc: 'north carolina',
+  mi: 'michigan', nj: 'new jersey', va: 'virginia', wa: 'washington', az: 'arizona', ma: 'massachusetts', tn: 'tennessee', in: 'indiana',
```

Biome (2.4.13)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)